modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/SW360MetaDataUpdater.java

```diff
@@ -33,36 +34,25 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(SW360MetaDataUpdater.class);
 
     // rest service adapters
-    private SW360AuthenticationClient authenticationClient;
     private SW360ProjectClientAdapter projectClientAdapter;
     private SW360LicenseClientAdapter licenseClientAdapter;
     private SW360ComponentClientAdapter componentClientAdapter;
     private SW360ReleaseClientAdapter releaseClientAdapter;
+    private SW360ConnectionConfiguration sw360ConnectionConfiguration;
 
-    private String userId;
-    private String password;
-    private String clientId;
-    private String clientPassword;
-
-    private boolean updateReleases = false;
-
-    public SW360MetaDataUpdater(String restServerUrl, String authServerUrl,
-                                String userId, String password, String clientId, String clientPassword,
-                                boolean proxyEnable, String proxyHost, int proxyPort) {
-        this.userId = userId;
-        this.password = password;
-        this.clientId = clientId;
-        this.clientPassword = clientPassword;
-
-        authenticationClient = new SW360AuthenticationClient(authServerUrl, proxyEnable, proxyHost, proxyPort);
-        projectClientAdapter = new SW360ProjectClientAdapter(restServerUrl, proxyEnable, proxyHost, proxyPort);
-        licenseClientAdapter = new SW360LicenseClientAdapter(restServerUrl, proxyEnable, proxyHost, proxyPort);
-        componentClientAdapter = new SW360ComponentClientAdapter(restServerUrl, proxyEnable, proxyHost, proxyPort);
-        releaseClientAdapter = new SW360ReleaseClientAdapter(restServerUrl, proxyEnable, proxyHost, proxyPort);
+    private boolean updateReleases;
+
+    public SW360MetaDataUpdater(SW360ConnectionConfiguration sw360ConnectionConfiguration, boolean updateReleases) {
```

Comment:
I believe `updateRelease` should be `uploadSources` here, since this is how it is called in the Updater